make/modules/java.base/Lib.gmk

```diff
@@ -227,7 +227,7 @@ ifeq ($(ENABLE_FALLBACK_LINKER), true)
       NAME := fallbackLinker, \
       CFLAGS := $(CFLAGS_JDKLIB) $(LIBFFI_CFLAGS), \
       LDFLAGS := $(LDFLAGS_JDKLIB) \
-                  $(call SET_SHARED_LIBRARY_ORIGIN), \
+                 $(call SET_SHARED_LIBRARY_ORIGIN), \
```

Comment:
If you are going to fix the indentation here, please do it to follow the code conventions for the build system:
https://openjdk.org/groups/build/doc/code-conventions.html
```
2. If a line must be broken, use four spaces for indentation.
18. Avoid padding internally in a line with spaces to try to align some feature into columns with surrounding lines.
```

Reply:
Hi Erik (@erikj79), 
BUILD_LIBFALLBACKLINKER is from different PR (#13079). If I understand correctly, for LIB_SIMD_SORT, are you suggesting that we don't pad the lines with spaces to align features into columns and instead just use 4 spaces for the indentation after the line break?

Reply:
I see now that this is an unrelated change. In that case please avoid changing whitespace in unrelated files for this PR.

Reply:
@erikj79 This space was inadvertently added as part of (https://github.com/openjdk/jdk/pull/14227) and @magicus requested to remove it in this follow-up PR.

Reply:
Ok, I missed that, never mind me then.